src/cogs/help.py

```diff
@@ -160,57 +93,55 @@ async def send_help(self, ctx, *args):
                 color=self.embed_color,
             )
 
-            for cog in cog_dict:
+            for cog in self.bot.cogs:
                 with suppress(Exception):  # forced to use this because pylint
                     embed.add_field(
                         name=cog,
-                        value=f'`{cog_dict[cog]["command_count"]} commands`',
+                        value="`N commands`",
```

Comment:
What is `N` here?

Reply:
Completely forgot to get rid of that! I just hadn't figured out how to count the enabled commands, I'll add that in tomorrow.